x/did/client/cli/tx.go

```diff
@@ -42,48 +37,29 @@ func GetCmdCreateDID(cdc *codec.Codec) *cobra.Command {
 			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
 			cliCtx := context.NewCLIContext().WithCodec(cdc).WithAccountDecoder(cdc)
 
-			privKey, err := genPrivKey(viper.GetBool(flagInteractive))
-			if err != nil {
-				return err
-			}
-
 			networkID, err := types.NewNetworkID(args[0])
 			if err != nil {
 				return err
 			}
 
-			pubKey := privKey.PubKey()
-			did := types.NewDID(networkID, pubKey, types.ES256K)
-			keyID := types.NewKeyID(did, "key1")
-			doc := types.NewDIDDocument(did, types.NewPubKey(keyID, types.ES256K, pubKey))
-
-			sig, err := types.Sign(doc, types.InitialSequence, privKey)
+			mnemonic, bip39Passphrase, err := readBIP39ParamsFrom(viper.GetBool(flagInteractive), client.BufferStdin())
 			if err != nil {
 				return err
 			}
-
-			passwd, err := client.GetCheckPassword(
-				"Enter a password to encrypt your key for DID to disk:",
-				"Repeat the password:",
-				client.BufferStdin(),
-			)
+			privKey, err := didcrypto.GenSecp256k1PrivKey(mnemonic, bip39Passphrase)
 			if err != nil {
 				return err
 			}
 
-			ks, err := keystore.NewKeyStore(keystoreBaseDir())
-			if err != nil {
-				return err
-			}
-			_, err = ks.Save(string(keyID), privKey[:], passwd)
```

Comment:
Moved this key storing logic to the new `savePrivKeyToKeyStore`, so that it can be a pair with `getPrivKeyFromKeyStore`.